### PR TITLE
fix golint for /pkg/volume/git_repo

### PR DESCRIPTION
**Pull request by mysunshine92**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind cleanup

**What this PR does / why we need it**:
fix golint for
pkg/volume/git_repo

**Which issue(s) this PR fixes**:

Fixes #
refer #68026
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/ok-to-test

Looks good to me modulo the test suite passes and also confirming that the change from `gitrepo` to `git_repo` was suggested by golint?

**Comment by mysunshine92** [comment]
> Looks good to me modulo the test suite passes and also confirming that the change from `gitrepo` to `git_repo` was suggested by golint?

yes, the change from gitrepo to git_repo was suggested by golint.

File: pkg/volume/gitrepo/git_repo_test.go [hunk 1]
```diff
@@ -113,14 +113,14 @@ func TestPlugin(t *testing.T) {
 				Name: "vol1",
 				VolumeSource: v1.VolumeSource{
 					GitRepo: &v1.GitRepoVolumeSource{
-						Repository: gitUrl,
+						Repository: gitURl,
```
Comment (mattjmcnaughton) [inline pkg/volume/gitrepo/git_repo_test.go]:
```suggestion
						Repository: gitURL,
```
I think you forgot to capitalize the final L :)

Comment (mattjmcnaughton) [inline pkg/volume/gitrepo/git_repo_test.go]:
```suggestion
						Repository: gitURL,
```
I think we missed capitalizing this final `L`.

**Review by mattjmcnaughton — COMMENTED** [review]
just one small change needed to fix the currently failing tests :)

File: pkg/volume/gitrepo/git_repo_test.go (hunk 1 above)
Comment (mysunshine92) [reply pkg/volume/gitrepo/git_repo_test.go]: thanks, i will modify it.

Comment (mysunshine92) [reply pkg/volume/gitrepo/git_repo_test.go]: @mattjmcnaughton thanks for your help ,i will modify it.

File: test/test_owners.csv [hunk 2]
```diff
@@ -816,7 +816,7 @@ k8s.io/kubernetes/pkg/volume/fc,rrati,0,
 k8s.io/kubernetes/pkg/volume/flexvolume,Q-Lee,1,
 k8s.io/kubernetes/pkg/volume/flocker,jbeda,1,
 k8s.io/kubernetes/pkg/volume/gcepd,saad-ali,0,
-k8s.io/kubernetes/pkg/volume/git_repo,davidopp,1,
+k8s.io/kubernetes/pkg/volume/gitrepo,davidopp,1,
```
Comment (msau42) [inline test/test_owners.csv]: Random question for @kubernetes/sig-testing , is this file even still maintained/used, or has it been obsoleted by owners files?  Looking at the storage related entries here, the owners are all mostly out of date...

Comment (BenTheElder) [reply test/test_owners.csv]: I don't think so, @spiffxp

Comment (spiffxp) [reply test/test_owners.csv]: it's used, not very well, but I'd rather see its usages replaced/removed before removing the file (ref: https://github.com/kubernetes/kubernetes/pull/81544#issuecomment-522256127)

File: pkg/volume/gitrepo/git_repo_test.go [hunk 3]
```diff
@@ -344,10 +343,9 @@ func doTestPlugin(scenario struct {
 		if os.IsNotExist(err) {
 			allErrs = append(allErrs,
 				fmt.Errorf("SetUp() failed, empty-dir wrapper path is not created: %s", podWrapperMetadataDir))
-		} else {
-			allErrs = append(allErrs,
-				fmt.Errorf("SetUp() failed: %v", err))
```
Comment (spiffxp) [inline pkg/volume/gitrepo/git_repo_test.go]: this should stay; unlike the else block that you removed above, there is no `return` in the if block

**Review by spiffxp — CHANGES_REQUESTED** [review]
one thing to change

**Pull request closed without merging**